packages/web-components/src/divider/divider.stories.ts

```diff
@@ -0,0 +1,128 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Divider as FluentDivider } from './divider.js';
+import { DividerAlignContent, DividerAppearance, DividerOrientation, DividerRole } from './divider.options.js';
+import './define.js';
+import SampleIcon from './paginated_reports_20_regular.svg';
+
+type DividerStoryArgs = Args & FluentDivider;
+type DividerStoryMeta = Meta<DividerStoryArgs>;
+
+const dividerTemplate = html<DividerStoryArgs>`
+  <fluent-divider
+    align-content=${x => x.alignContent}
+    appearance=${x => x.appearance}
+    role=${x => x.role}
+    ?inset=${x => x.inset}
+    orientation=${x => x.orientation}
+    empty=${x => (x.content ? null : '')}
```

Comment:
@chrisdholt I would like to move this into the template, or the component. This checks to see if the slot has content and if not, apply an attribute: empty. I leverage this attribute selector in my CSS to set the min-height of the pseudo elements. I'm open to suggestions.

Reply:
Got it! Thanks for the help.